go/test/endtoend/cluster/vtctlclient_process.go

```diff
@@ -47,8 +47,16 @@ func (vtctlclient *VtctlClientProcess) InitShardPrimary(Keyspace string, Shard s
 	// version_upgrade test depends on using older binaries
 	// which means we cannot use the new InitShardPrimary command here
 	// TODO(deepthi): fix after v12.0
+	version, err := GetMajorVersion("vttablet")
+	if err != nil {
+		return err
+	}
+	cmd := "InitShardMaster"
+	if version > 12 {
+		cmd = "InitShardPrimary"
+	}
 	output, err := vtctlclient.ExecuteCommandWithOutput(
-		"InitShardMaster",
+		cmd,
```

Comment:
This change was required to unblock the upgrade/downgrade tests. We used to execute v13.0.0 tests using the v12 codebase, and `InitShardMaster` does not exist in v13.0.0